openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -1998,18 +1998,13 @@ namespace detail {
   {
     boost::optional<AirLoopHVACSupplyPlenum> result;
 
-    std::vector<AirLoopHVACSupplyPlenum> plenums = model().getModelObjects<AirLoopHVACSupplyPlenum>();
+    std::vector<AirLoopHVACSupplyPlenum> plenums = getObject<ModelObject>().getModelObjectSources<AirLoopHVACSupplyPlenum>();
```

Comment:
@macumber should use `model().getConcreteModelObjects<AirLoopHVACSupplyPlenum>()` instead, it'll be higher performance and is probably driving most of slowness here.